src/components/Loading/Loading-story.js

```diff
@@ -8,13 +8,14 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 
-import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { withKnobs, boolean, text } from '@storybook/addon-knobs';
 import Loading from '../Loading';
 
 const props = () => ({
   active: boolean('Active (active)', true),
   withOverlay: boolean('With overlay (withOverlay)', false),
   small: boolean('Small (small)', false),
+  description: text('Description (description)', 'Active loading indicator'),
```

Comment:
This doesn't need to be a knob in my opinion. With a descriptive prop-type comment, there's no reason someone would need to 'see' what their own svg title would look like.